mora/service/engagement.py

```diff
@@ -98,7 +98,7 @@ def list_details(type, id):
     '/<any("e", "ou"):type>/<uuid:id>/details/<function>',
 )
 @util.restrictargs('at', 'validity', 'start', 'limit')
-def get_engagement(type, id, function):
+def get_detail(type, id, function):
```

Comment:
I think it's actually time to move this to a file that isn't `engagement.py`

Reply:
Really? What a novel suggestion 😏